package-lock.json

```diff
@@ -2329,7 +2329,8 @@
         "ansi-regex": {
```

Comment:
Please remove this file from the commit.

Reply:
Done, sorry I didn't take note of this before